packages/grafana-schema/src/common/common.gen.ts

```diff
@@ -640,6 +649,7 @@ export interface TableImageCellOptions {
 export interface TableBarGaugeCellOptions {
   mode?: BarGaugeDisplayMode;
   type: TableCellDisplayMode.Gauge;
+  valueMode?: BarGaugeValueMode;
```

Comment:
More of a naming nit: I think it would be more consistent to have `mode` renamed to `displayMode` alongside `valueMode`, or maybe rename `valueMode` to `valueDisplay`? Not entirely sure, but I feel it's a bit confusing right now.

Reply:
I think you're right about this, probably best to call it `valueDisplayMode` or something similar.

Reply:
Yes, or that!